course/src/main/java/org/openedx/course/presentation/ui/CourseUI.kt

```diff
@@ -110,89 +98,6 @@ import subtitleFile.TimedTextObject
 import java.util.Date
 import org.openedx.core.R as coreR
 
-@Composable
-fun CourseImageHeader(
-    modifier: Modifier,
-    apiHostUrl: String,
-    courseImage: String?,
-    courseCertificate: Certificate?,
-    onCertificateClick: (String) -> Unit = {},
-    courseName: String,
-) {
-    val configuration = LocalConfiguration.current
-    val windowSize = rememberWindowSize()
-    val contentScale =
-        if (!windowSize.isTablet && configuration.orientation == Configuration.ORIENTATION_LANDSCAPE) {
-            ContentScale.Fit
-        } else {
-            ContentScale.Crop
-        }
-    val imageUrl = if (courseImage?.isLinkValid() == true) {
-        courseImage
-    } else {
-        apiHostUrl.dropLast(1) + courseImage
-    }
-    Box(modifier = modifier, contentAlignment = Alignment.Center) {
-        AsyncImage(
-            model = ImageRequest.Builder(LocalContext.current)
-                .data(imageUrl)
-                .error(coreR.drawable.core_no_image_course)
-                .placeholder(coreR.drawable.core_no_image_course)
-                .build(),
-            contentDescription = stringResource(
-                id = coreR.string.core_accessibility_header_image_for,
-                courseName
-            ),
-            contentScale = contentScale,
-            modifier = Modifier
-                .fillMaxSize()
-                .clip(MaterialTheme.appShapes.cardShape)
-        )
-        if (courseCertificate?.isCertificateEarned() == true) {
-            Column(
-                Modifier
-                    .fillMaxSize()
-                    .clip(MaterialTheme.appShapes.cardShape)
-                    .background(MaterialTheme.appColors.certificateForeground),
-                horizontalAlignment = Alignment.CenterHorizontally,
-                verticalArrangement = Arrangement.Center
-            ) {
-                Icon(
-                    modifier = Modifier.testTag("ic_congratulations"),
-                    painter = painterResource(id = R.drawable.ic_course_completed_mark),
```

Comment:
We can remove the drawable as well.

Reply:
That makes sense; thank you! ✅

Reply:
@HamzaIsrar12, could you please re-approve after the change?

Reply:
Done ✨